bucket/gitextensions.json

```diff
@@ -17,6 +17,7 @@
             "Git Extensions"
         ]
     ],
+    "persist": "GitExtensions.settings",
```

Comment:
This file does not exists in archive. Add pre_install block with persistance check and file creatiing